charts/fluent-operator/templates/fluentbit-clusterinput-tail.yaml

```diff
@@ -25,6 +25,8 @@ spec:
     skipLongLines: {{ .Values.fluentbit.input.tail.skipLongLines }}
     db: /fluent-bit/tail/pos.db
     dbSync: Normal
+    storageType: {{ .Values.fluentbit.input.tail.storageType }}
+    pauseOnChunksOverlimit: {{ .Values.fluentbit.input.tail.pauseOnChunksOverlimit }}
```

Comment:
This field is only functional when the storage type is `fileSystem` right? If that's the case then maybe add a check for this field to see what the storage type is

Reply:
Although it will by default off and also no effect if it is enabled with `fileSystem` as storage. Still I have added check on this. Its always better to have one check on higher level.